zerver/tests/webhooks/test_mention.py

```diff
@@ -0,0 +1,26 @@
+# -*- coding: utf-8 -*-
+from six import text_type
+from zerver.lib.test_classes import WebhookTestCase
+
+class MentionHookTests(WebhookTestCase):
+    STREAM_NAME = 'test'
+    URL_TEMPLATE = "/api/v1/external/mention?api_key={api_key}&stream={stream}"
+    FIXTURE_DIR_NAME = 'mention'
+
+    def test_mention_webfeed(self):
+        # type: () -> None
+        expected_topic = u"news"
+        expected_message = u"**[Historical Sexual Abuse (Football): 29 Nov 2016: House of Commons debates - TheyWorkForYou]"
+                            "(https://www.theyworkforyou.com/debates/?id=2016-11-29b.1398.7&p=24887)**:\n"
+                            "\u2026 Culture, Media and Sport\nNothing is more important than keeping "
+                            "children safe. Child sex abuse is an exceptionally vile crime, and all of "
+                            "Government take it very seriously indeed, as I know this House does."
+                            "\nChildren up and down the country are \u2026"
```

Comment:
It should automatically concatenate the strings together. I just forgot to put it in parentheses. That should correct the tests.